src/components/input/bl-input.css

```diff
@@ -84,6 +89,16 @@ input {
   background-color: transparent;
 }
 
+input:disabled {
+  cursor: not-allowed;
+}
+
+input:-webkit-autofill,
```

Comment:
I think we need to use `:autofill` as well. It's supported by Firefox and Safari. Most probably vendor prefix will drop in the future.